scripts/categorizer.py

```diff
@@ -6,12 +6,81 @@ def find_empty_lines(line):
     :param line:
     :return:
     """
+
     if line.strip("\t").strip(" ").strip("\n") == '':
         return 1
 
     return 0
 
+def find_comment_lines(lines):
+    """
+    Tries to categorize comment lines.
+    We treat comment lines as non-code or something that can
+    cause false positives for other categorizers.
+
+    Also comments + multi line is a double categorization.
+    We are trying to keep categorizations mostly separate from each other,
+    but this one breaks that rule.
```

Comment:
this to "the comment categorization breaks that rule." It categorizes multi-line and comments at the same time.